gulpfile.js

```diff
@@ -22,10 +22,10 @@ var reload = browserSync.reload;
 gulp.task('browser-sync', function() {
   browserSync.init({
     server: {
-      baseDir: ["./examples", "./dev"]
+      baseDir: ["./examples", "./dev", "./dist"]
```

Comment:
Right right - I wanted to use ``/dev`` as well, however, I didn't know which file to include in my ``example`` directory as using ``<script src="../dev/maptable.js"></script>`` did not work. 

Which ``maptable.js`` file should I include in my ``example/index.html`` file such that I get the re-compiled version reloaded every time I edit it?

 Again, I suspect this has to me being an npm/bower/gulp newbie!